CMakeLists.txt

```diff
@@ -82,8 +84,10 @@ function(setup_target TARGET)
   endif()
   # Allow RDKit includes to be prefixed
```

Comment:
It seems worth adding a "and add all of the relevant Sketcher-specific include directories" or something like that to this comment.  I was really confused for a few seconds as to how `${PROJECT_BINARY_DIR}/include` related to RDKit includes.